### PR TITLE
move the logic for copying in IV to the mandataris service

### PR DESCRIPTION
**Pull request by Rahien**
## Description

Moves the logic for mirroring draft mandataris instances during the IV to the mandataris service. That way we don't have O(N) http calls that can all fail independently, causing inconsistent state.

## How to test

Use the test process defined in the linked pr

## Links to other PR's

- https://github.com/lblod/frontend-lokaal-mandatenbeheer/pull/367

### CODE REVIEW

File: routes/intallatievergadering.ts [hunk 1]
```diff
@@ -333,4 +349,234 @@ async function movePersons(installatievergaderingId: string) {
...
+) {
+  await clearMandatarisInstancesFromOrgaan(orgaanItTo);
+  await constructNewMandatarisInstances(orgaanItFrom, orgaanItTo);
+}
+
+async function constructNewMandatarisInstances(
+  orgaanItFrom: string,
+  orgaanItTo: string,
+) {
+  // TODO test this
```
Comment (bfidlers) [inline routes/intallatievergadering.ts:364]: You still have a todo here.

**Review by bfidlers — COMMENTED** [review]
I believe this logic does belongs here, so that is better. But I'm finding it a bit difficult to read. 

Also I feel like we lose some protection we did get in the frontend, such as a check if you don't have mandatarissen to sync, then it does not delete the mandatarissen from the OCMW bestuursorgaan and you get a warning etc.

File: routes/intallatievergadering.ts [hunk 2]
```diff
@@ -333,4 +349,272 @@ async function movePersons(installatievergaderingId: string) {
   await updateSudo(sparql);
 }
 
+async function copyCommuneMandatarisInstancesToOCMW(
```
Comment (bfidlers) [inline routes/intallatievergadering.ts:352]: If you want to specify from where to where you copy, I would use municipality, that is the more often used translation for "gemeente".

File: routes/intallatievergadering.ts [hunk 3]
```diff
@@ -333,4 +349,272 @@ async function movePersons(installatievergaderingId: string) {
...
+    // voorzitter gemeenteraad -> voorzitter raad voor maatschappelijk welzijn
+    'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000012':
+      'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000016',
+    // schepen -> lid vast bureau
+    'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000014':
+      'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000017',
+    // aangewezen burgemeester -> voorzitter vast bureau
+    'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/7b038cc40bba10bec833ecfe6f15bc7a':
+      'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000018',
+  };
```
Comment (bfidlers) [inline routes/intallatievergadering.ts:398]: There are some constants defined in `controllers\linked-mandataris.ts`, might be a good idea to extract those and not have this duplication?

Comment (bfidlers) [reply routes/intallatievergadering.ts:398]: Hmmm unfortunately the burgemeester is different ...

**Review by bfidlers — APPROVED** [review]
Makes it a bit easier to understand.

Mandataris-service is still a bit of the wild west, but we need to look into that at a later stage.

File: routes/intallatievergadering.ts (hunk 3 above)
Comment (Rahien) [reply routes/intallatievergadering.ts]: good point about burgemeester being differnet, i think in the linking part it is important to do both, in the iv only the aangewezen burgemeester will count

**Pull request merged**